include/mfmg/cuda/cuda_mesh_evaluator.cuh

```diff
@@ -32,9 +32,9 @@ public:
                     dealii::DoFHandler<dim> &dof_handler,
                     dealii::AffineConstraints<double> &constraints);
 
-  int get_dim() const final;
+  virtual int get_dim() const override final;
```

Comment:
Why isn't this `static`?

Reply:
If this function is `static`, it can't be inherited. This is possible but that means introducing more template in `hierarchy.hpp` to get ride of inheritance.

Reply:
I see. Let's make that an issue and move on.

Reply:
See #210.